src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts

```diff
@@ -417,6 +421,33 @@ export class AcrossConfigStoreClient extends BaseAbstractClient {
         if (!isNaN(args.value)) {
           this.cumulativeMaxRefundCountUpdates.push(args);
         }
+      } else if (args.key === utf8ToHex(GLOBAL_CONFIG_STORE_KEYS.CHAIN_ID_INDICES)) {
+        // First remove out all the spaces
+        const rawChainIndices = String(args.value).replace(/\s/g, "");
```

Comment:
Instead of removing spaces can't we just `JSON.parse(args.value)` and then check that its a list of positive integers including the chain ID 1? Said another way, why isn't the regex check a superfluous check?

Reply:
Resolved offline